src/libsyncengine/syncpal/syncpalworker.cpp

```diff
@@ -358,16 +367,46 @@ void SyncPalWorker::initStep(SyncStep step, std::shared_ptr<ISyncWorker> (&worke
     }
 }
 
+SentryHandler::PTraceName SyncPalWorker::setpToPTraceName(SyncStep step) const {
```

Comment:
Perhaps this could be defined in `SentryHandler` instead to avoid increasing the size of `SyncPalWorker` class

Reply:
Done here:[27018de](https://github.com/Infomaniak/desktop-kDrive/pull/403/commits/27018dec3ca712b4786562a19ba2372340616af0)
To avoid a >1k lines sentry handler, I also separated it in several files.